dev/dags/dbt/jaffle_shop/succeed.txt

```diff
@@ -0,0 +1,127 @@
+adal==1.2.7
```

Comment:
similarly can we please add a comment describing the purpose of this file?

Reply:
We don't need this, it's a left over, deleting it!

Reply:
Remove them. They were left over files from previous tests! Thanks for pointing out